src/ggrc/assets/javascripts/controllers/tree_view_controller.js

```diff
@@ -1445,12 +1449,17 @@ can.Control("CMS.Controllers.TreeViewNode", {
   }
 
   , draw_node: function() {
+    if (this._draw_node_in_progress) {
```

Comment:
Ah, yes, I didn't know it gets resolved once the placeholder is loaded. A separate var is a must.